src/mesh/mesh.cpp

```diff
@@ -87,6 +88,7 @@ Mesh::Mesh(ParameterInput *pin, Properties_t &properties, Packages_t &packages,
       // private members:
       next_phys_id_(),
       num_mesh_threads_(pin->GetOrAddInteger("parthenon/mesh", "num_threads", 1)),
+      num_mesh_streams_(pin->GetOrAddInteger("parthenon/mesh", "num_streams", 1)),
```

Comment:
Seems like the default should be higher?

Reply:
Maybe. I'm currently undecided and mainly set it to one as a "safe" default which results in no extra execution space being created (or destroyed).
I could see us changing this once we have more information what a "reasonable" number is (potentially even taking the MeshBlock size into account when setting a default value).

Reply:
Can we add a `// TODO` there summarizing this conversation? That way we won't forget.

Reply:
done